client/js/components/procedure/StatementSegmentsList/StatementMeta/StatementPublicationAndVoting.vue

```diff
@@ -43,7 +43,7 @@ All rights reserved
       :has-permission-to-edit="editable && statement.attributes.isManual"
       :translation-keys="translationKeys"
       ref="listComponent"
-      @reset="resetForm()"
+      @delete="handleDeleteVote"
```

Comment:
I can't find the handleDeleteVote method

Reply:
thanks. I forgot to delete that. It was just a test

Reply:
Done here [f029efe](https://github.com/demos-europe/demosplan-core/pull/3996/commits/f029efe44d723cd2152cf60d669be150b247210e)